src/solutions/how-tos/confluent-cloud/iot-ingest-control.md

```diff
@@ -0,0 +1,455 @@
+
+```bash:no-line-numbers
+cat /var/log/cloud-init-output.log
+```
+
+```output:no-line-numbers
+Cloud-init v. 22.2.2 running 'init'...
+```
+
+:::
+
+### Configure Global DNS
+
+> This ensures that any new Kafka brokers added to the Confluent Cloud cluster can still be reached via the <ZillaPlus/> proxy.
+
+When using a wildcard DNS name for your own domain, such as `*.example.aklivity.io` then the DNS entries are setup in your DNS provider.
+
+Navigate to the [CloudFormation console](https://console.aws.amazon.com/cloudformation). Then select the `my-zilla-iot-proxy` stack to show the details.
+
+::: note Check your selected region
+Make sure you have selected the desired region, such as `US East (N. Virginia) us-east-1`.
+:::
+
+In the stack `Outputs` tab, find the public DNS name of the `NetworkLoadBalancer.`
+
+You need to create a `CNAME` record mapping your public DNS wildcard pattern to the public DNS name of the Network Load Balancer.
+
+::: info
+You might prefer to use an Elastic IP address for each NLB public subnet, providing DNS targets for your `CNAME` record that can remain stable even after restarting the stack.
+:::
+
+## Verify MQTT Client Connectivity
+
+To verify that we have successfully enabled public internet connectivity to our Confluent Cloud cluster from the local development environment, we will use a generic Kafka client to create a topic, publish messages and then subscribe to receive these messages from our Confluent Cloud cluster via the public internet.
+
+### Connect with an MQTT Client
+
+> This verifies internet connectivity to your Confluent Cloud cluster via Zilla Plus for Confluent Cloud.
+
+We can now verify that the Kafka client can successfully communicate with your Confluent Cloud cluster via the internet from your local development environment to create a topic, then publish and subscribe to the same topic.
+
+::: warning
+Replace these TLS bootstrap server names accordingly for your own custom wildcard DNS pattern.
+:::
+
+### Subscribe to a topic
+
+Publish two messages to the newly created topic via the following producer command:
+
+```bash:no-line-numbers
+bin/kafka-console-producer.sh \
+  --topic zilla-plus-test \
+  --producer.config confluent.properties \
+  --broker-list <tls-bootstrap-server-names>
+```
+
+A prompt will appear for you to type in the messages:
+
+```output:no-line-numbers
+>This is my first event
+>This is my second event
+```
+
+### Publish to a topic
+
+Read these messages back via the following consumer command:
+
+```bash:no-line-numbers
+bin/kafka-console-consumer.sh \
+  --topic zilla-plus-test \
+  --from-beginning \
+  --consumer.config confluent.properties \
+  --bootstrap-server <tls-bootstrap-server-names>
+```
+
+You should see the `This is my first event` and `This is my second event` messages.
+
+```output:no-line-numbers
+This is my first event
+This is my second event
+```
```

Comment:
This section needs to be use an MQTT client like `mosquitto` instead of a Kafka client.